### PR TITLE
Add calculator component

### PR DESCRIPTION
**Pull request by Metsanpeitto**
In this branch I have followed these steps:
- :wastebasket:  Delete all the boilerplate from CRA (text, images, styles).
- :file_folder:  Create a directory called `components`.
- :new: Inside `components`, create a new `Calculator.js` file.
- :art:  In `Calculator.js`, create a React component that matches the given design.
-  :no_good:  Don't add any extra design.
- :paintbrush:  Develop just the UI of the calculator.
- :tophat:  Use class-based components.
- :1234:  Once the Calculator is ready, import it and use it in your main component (App.js).
-:test_tube:  Test it in the browser.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```
Comment (arslanbisharat) [inline src/App.js]: - [x] App.js should have a `class-based` component as present in **src/compnenets/calculator.js**.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+class Calculator extends React.Component {
```
Comment (arslanbisharat) [inline src/components/Calculator.js:3]: - [ ] App js should have imported calculator.js

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 1 above)
Comment (Metsanpeitto) [reply src/App.js]: Thanks for your time @Arslan. I would love to implement those changes. However, I don't understand what you mean. Could you please guide me to your explanation so I can clarify my doubts?

#### 1st Doubt 
The snippet code that you highlight there doesn't belong to my app or at least in my browser the rendered app architecture looks like follows :
```
<div class="App"><div class="calculator-body"><div class="body-result"><h5>0</h5></div><div class="body-buttons"><div class="buttons-row-1"><button type="button" class="col-1 ">AC</button><button type="button" class="col-2 ">+/-</button><button type="button" class="col-3 ">%</button><button type="button" class="col-4 orange">/</button></div><div class="buttons-row-2"><button type="button" class="col-1 ">7</button><button type="button" class="col-2 ">8</button><button type="button" class="col-3 ">9</button><button type="button" class="col-4 orange">x</button></div><div class="buttons-row-3"><button type="button" class="col-1 ">4</button><button type="button" class="col-2 ">5</button><button type="button" class="col-3 ">6</button><button type="button" class="col-4 orange">-</button></div><div class="buttons-row-4"><button type="button" class="col-1 ">1</button><button 
```

File: src/components/Calculator.js (hunk 2 above)
Comment (Metsanpeitto) [reply src/components/Calculator.js:3]: #### 2nd Doubt 

I don't understand the purpose or function of importing App.js into Calculator.js to then import Calculator.js into App.js.

Could you please expand on your argument? :confused:

File: src/App.js (hunk 1 above)
Comment (arslanbisharat) [reply src/App.js]: What I meant is please change app.js code to class-based.

File: src/components/Calculator.js (hunk 2 above)
Comment (arslanbisharat) [reply src/components/Calculator.js:3]: I didn't say you import calculator.js to app.js
Please make sure you have imported app.js to calculator.js for future purposes.

File: src/App.js (hunk 1 above)
Comment (Metsanpeitto) [reply src/App.js]: :heavy_check_mark:  Understood

File: src/components/Calculator.js (hunk 2 above)
Comment (Metsanpeitto) [reply src/components/Calculator.js:3]: :warning:  DISCLAIMER TO THE NEW CODE REVIEVWER :warning:
```
I didn't say you import calculator.js to app.js
Please make sure you have imported app.js to calculator.js for future purposes.
```
:rotating_light:  This last requirement of the last code reviewer was a mistake on his part, so please omit it :rotating_light:

File: src/App.js (hunk 1 above)
Comment (Metsanpeitto) [reply src/App.js]: :heavy_check_mark: 1st Request : " What I meant is please change app.js code to class-based."

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @Metsanpeitto , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Metsanpeitto** [comment]
Thanks, @DeliceLydia , you are awesome :) :gift:

**Pull request merged**